### PR TITLE
feat(metrics): pass the reminder param to account verification

### PR DESCRIPTION
**Pull request by vladikoff**
Used with https://github.com/mozilla/fxa-auth-server/commit/aca4185a826a70e5fe57247490238e63ddce5ce0

### CODE REVIEW

**Comment by vladikoff** [comment]
@vbudhram r?

**Comment by vladikoff** [comment]
Used with https://github.com/mozilla/fxa-content-server/issues/3864

File: tests/lib/verifyCode.js [hunk 1]
```diff
@@ -123,6 +123,34 @@ define([
...
+            uid = result.uid;
+            assert.ok(uid, 'uid is returned');
+
+            return respond(mail.wait(user), RequestMocks.mail);
+          })
+          .then(function (emails) {
+            var code = emails[0].html.match(/code=([A-Za-z0-9]+)/)[1];
+            assert.ok(code, 'code is returned');
+
+            return respond(client.verifyCode(uid, code, { reminder: 'first' }),
```
Comment (vbudhram) [inline tests/lib/verifyCode.js]: I know this is in hindsight, but would it make sense to be strict on the auth-server for the value of reminder by making the only accepted values, `first` and `second`? Currently, with a [random reminder value](https://github.com/mozilla/fxa-auth-server/blob/master/lib/routes/account.js#L1336), auth-server will still verify account but won't log anything. Likewise, validation could be added on the js-client to ensure only these values are processed.

Comment (vladikoff) [inline tests/lib/verifyCode.js]: @vbudhram I did not validate 'first' and 'second' because we might add other types of reminders in the future.

Comment (vladikoff) [inline tests/lib/verifyCode.js]: or maybe A/B test different types of emails (with different reminder names)

**Comment by vbudhram** [comment]
@vladikoff Just the minor note, not sure if it is worthwhile to add in, otherwise r+

**Pull request merged**